### PR TITLE
Use dotnetcli and markdown fixes

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/29edfd8e252bd46d7984cd1438894e298e128fa7/articles/install-guide/update.md) | 87 | 66 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e196f52-3a9e-4e61-a57d-5c86ab073c37_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0a367c56f94718d7a12da82d9055699649db2ddd/articles/install-guide/index.md) | 90 | 81 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/30b3bf65-3f9b-406c-aaa3-cfcda7836ee9_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0a367c56f94718d7a12da82d9055699649db2ddd/articles/install-guide/update.md) | 87 | 66 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c47a0e6-43bb-4a8e-950a-cdbcd8a853a0_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8a49987f33ce53d2b3c41e74a6ad5941e174d0e1/articles/how-to-guides/create-qsharp-project.md) | 92 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d017dd87-e99e-401e-8623-9245e0cb0e6d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8a49987f33ce53d2b3c41e74a6ad5941e174d0e1/articles/install-guide/index.md) | 90 | 81 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f981f2a-6fb6-430e-a5a0-d7db90104f9e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8a49987f33ce53d2b3c41e74a6ad5941e174d0e1/articles/install-guide/update.md) | 87 | 66 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/caa69fd7-78e5-47e5-9f8d-1968590cda25_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/08223285f66269c02cfd34266d629e9d89d2ce98/articles/how-to-guides/create-qsharp-project.md) | 92 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6bab1f0a-1e4d-496f-8838-615069269226_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/08223285f66269c02cfd34266d629e9d89d2ce98/articles/install-guide/index.md) | 90 | 81 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da32736b-3458-4659-931e-8413c4fb9db5_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/08223285f66269c02cfd34266d629e9d89d2ce98/articles/install-guide/update.md) | 87 | 66 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89f63044-7eef-4a5c-a884-f8c4bba4299a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/08223285f66269c02cfd34266d629e9d89d2ce98/articles/libraries/numerics/installation.md) | 82 | 45 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/601d8f7e-d3c6-4416-a9e0-a40064aee708_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/908edf6e2d6d3d7fc5223fad3a496b312246a39f/articles/how-to-guides/create-qsharp-project.md) | 92 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1542eda9-c144-453d-9233-cdb43c520c4e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/908edf6e2d6d3d7fc5223fad3a496b312246a39f/articles/install-guide/index.md) | 90 | 81 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/365738bb-c82c-43e0-968f-4b509796fa76_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/908edf6e2d6d3d7fc5223fad3a496b312246a39f/articles/install-guide/update.md) | 87 | 66 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f893417f-cf12-4579-8781-d642c189e523_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/908edf6e2d6d3d7fc5223fad3a496b312246a39f/articles/libraries/numerics/installation.md) | 82 | 45 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/63cd20ee-b90f-4616-b774-3e7044726144_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d82674a77dfa12fbf034cc6cbb8d00454681ca04/articles/how-to-guides/create-qsharp-project.md) | 92 | 72 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/652c4d0b-ebc7-4d8b-8aec-b351af8fc95c_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d82674a77dfa12fbf034cc6cbb8d00454681ca04/articles/install-guide/index.md) | 90 | 81 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/284d49ad-7144-403c-8c14-369679866646_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d82674a77dfa12fbf034cc6cbb8d00454681ca04/articles/install-guide/update.md) | 87 | 66 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0702920b-1edf-41f3-8a9a-0bba1e91a8f8_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d82674a77dfa12fbf034cc6cbb8d00454681ca04/articles/libraries/chemistry/installation.md) | 74 | 47 | 47 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bedaf815-cce4-4a4e-a877-3bc35063548b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d82674a77dfa12fbf034cc6cbb8d00454681ca04/articles/libraries/numerics/installation.md) | 82 | 45 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a03b5272-0df4-4903-aaac-fbb3d9d071c4_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/update.md [hunk 1]
```diff
@@ -18,10 +18,12 @@ This article assumes that you already have the QDK installed. If you are install
...
 
 1. First, install the latest version of the [.NET Core SDK 3.0](https://dotnet.microsoft.com/download) and run the following command in the command prompt:
-```bash
+
+```dotnetcli
 dotnet --version
 ```
- Verify the output is 3.0.100 or higher, then follow the instructions below depending on your setup.
+
+Verify the output is 3.0.100 or higher, then follow the instructions below depending on your setup.
```
Comment (bettinaheim) [inline articles/install-guide/update.md]: While we are at it, I'd suggest to update this to 3.1.100. 
```suggestion
Verify the output is 3.1.100 or higher, then follow the instructions below depending on your setup.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62ae3c5d525d46f58502e5ae87561d212110e06f/articles/how-to-guides/create-qsharp-project.md) | 90 | 72 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba07dd56-a7f6-4527-86b3-c48ac9d00657_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62ae3c5d525d46f58502e5ae87561d212110e06f/articles/install-guide/update.md) | 84 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33f6578a-2565-4d37-b25e-87335cb592d2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62ae3c5d525d46f58502e5ae87561d212110e06f/articles/libraries/chemistry/installation.md) | 74 | 47 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/836c4bf3-2250-4977-b0b5-cb2451ea4fed_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/62ae3c5d525d46f58502e5ae87561d212110e06f/articles/libraries/numerics/installation.md) | 82 | 45 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d2b33e20-9d02-4ad4-b6f7-f0bd228c075c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0db0993aa18151b24b84045b98e407f4741a1222/articles/how-to-guides/create-qsharp-project.md) | 90 | 72 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/67431622-dd18-425d-8df7-ed31a60aed9d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0db0993aa18151b24b84045b98e407f4741a1222/articles/install-guide/update.md) | 84 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a8340f5-9671-4d94-8299-61dbcf8000ca_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0db0993aa18151b24b84045b98e407f4741a1222/articles/libraries/chemistry/installation.md) | 74 | 47 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d9e2586e-762f-44f5-95be-64f2ddca2e5a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/0db0993aa18151b24b84045b98e407f4741a1222/articles/libraries/numerics/installation.md) | 82 | 45 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fca465c7-4be4-4a8a-9255-b54b20f2879a_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/667edf0ac84c68954fed0a8faf87a4a73fd165b9/articles/how-to-guides/create-qsharp-project.md) | 90 | 72 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8ae5b09-029e-4f8a-8c37-4dbf8e680e11_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/667edf0ac84c68954fed0a8faf87a4a73fd165b9/articles/install-guide/update.md) | 84 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/766fe529-0259-4e94-9a6b-d182715e1f27_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/667edf0ac84c68954fed0a8faf87a4a73fd165b9/articles/libraries/chemistry/installation.md) | 74 | 47 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/71332e2c-e942-4c14-a093-abc19f6cfce3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/667edf0ac84c68954fed0a8faf87a4a73fd165b9/articles/libraries/numerics/installation.md) | 82 | 45 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e343b1e4-f9fe-4240-9d96-d09f902a0cb0_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/how-to-guides/create-qsharp-project.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4f956390658b5c0886a57e206a7054c19b357161/articles/how-to-guides/create-qsharp-project.md) | 90 | 72 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca7a433b-f651-4fe2-a403-91cb983f19ce_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/csinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4f956390658b5c0886a57e206a7054c19b357161/articles/install-guide/csinstall.md) | 92 | 64 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f9bd0f8c-a645-447f-b9fa-e219b7075dd1_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4f956390658b5c0886a57e206a7054c19b357161/articles/install-guide/update.md) | 84 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0dd31e97-f980-4a2a-8c66-de99997d9dcc_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/chemistry/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4f956390658b5c0886a57e206a7054c19b357161/articles/libraries/chemistry/installation.md) | 74 | 47 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/31111d1a-d5e6-4a9a-807a-a8d38daf5005_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/numerics/installation.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4f956390658b5c0886a57e206a7054c19b357161/articles/libraries/numerics/installation.md) | 82 | 45 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c65ef49f-4db5-4289-a922-e2406864ac68_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**